### PR TITLE
Optimizing website builder image

### PR DESCRIPTION
**Pull request by uidoyen**

### CODE REVIEW

File: website-builder/Dockerfile [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
-    && apt-get install -y nodejs
+ENV SITE_HOME=/site
 
-# Install Yarn
-RUN npm install -g yarn
+RUN apk add --no-cache build-base gcc bash cmake git jq nodejs yarn
 
 # Install Antora framework
-RUN yarn global add @antora/cli@2.3.4 @antora/site-generator-default@2.3.4 \
+RUN yarn global add @antora/cli @antora/site-generator-default \
```
Comment (gunnarmorling) [inline website-builder/Dockerfile:10]: Can we keep the versions? Always better to explicitly specify them, so to not suffer from any breaking updates.

File: website-builder/Dockerfile [hunk 2]
```diff
@@ -1,38 +1,35 @@
...
-
 # Install Rake and Bundler. This is the minimum needed to generate the site ...
-RUN gem install rdoc -v 6.2.0
-RUN gem install rake bundler
-RUN gem install bundler
-RUN gem install jekyll
- 
+RUN gem install bundler -v "~>1.0" \
+    && gem install rdoc -v 6.2.0 \
+    && gem install bundler jekyll \
```
Comment (gunnarmorling) [inline website-builder/Dockerfile:23]: Would be great to pin versions here, too.

**Review by gunnarmorling — COMMENTED** [review]
Nice one! I take it you could build the website, including docs, locally using this image?

File: website-builder/Dockerfile (hunk 1 above)
Comment (uidoyen) [reply website-builder/Dockerfile]: Added it back. Yes, It works locally. I tested with `uidoyen/newjekyll`.

File: website-builder/Dockerfile (hunk 2 above)
Comment (uidoyen) [reply website-builder/Dockerfile]: Pinned Jekyll version as per `Gemfile`

**Comment by gunnarmorling** [comment]
Thx, @uidoyen !

**Pull request merged**